architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -432,4 +432,39 @@ to run:
 `oc delete istag/ruby:latest`
 ====
 
+[[insecure-registries]]
 
+=== Importing images from insecure registries
+An image stream can be configured to import tag and image metadata from an image
+repository that is signed with a self-signed certificate. To do that, add
+`*openshift.io/image.insecureRepository*` annotation set to `*true*`. This setting
+will bypass certificate validation when connecting to registry.
+
+====
+
+[source,json]
+----
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "ruby",
+    "annotations": {
+      "openshift.io/image.insecureRepository": "true"
+    }
+  },
+  "spec": {
+    "dockerImageRepository": "my.repo.com:5000/myimage"
+  }
+}
+----
+====
+
+[IMPORTANT]
+====
+The above definition will only apply to the image controller which is responsible
+for retrieving image metadata. For this image to be used in the cluster each
+node needs to have docker configured with `--insecure-registry` flag. See
+link:../../install_config/install/docker_registry.html#securing-the-registry[securing the registry]
```

Comment:
I wouldn't link to this. This link is about the integrated registry. You could say that if you want Docker to be able to pull from or push to the registry in question, you'll need to also use the --insecure-registry setting for Docker for that registry. You could maybe link to the prerequisites page, which describes how to set up --insecure-registry for the cluster subnet, but make sure it's clear that you need to mark this registry insecure.